data/assets/scene/solarsystem/heliosphere/todayssun/grid.asset

```diff
@@ -0,0 +1,118 @@
+local transforms = asset.require("scene/solarsystem/sun/transforms_heliosphere")
+local sunTransforms = asset.require("scene/solarsystem/sun/transforms")
+
+
+
+local size = 700000000
+
+local carrington_prime_meridian = {
+  Identifier = "CarringtonPrimeMeridian",
+  Parent = sunTransforms.SunIAU.Identifier,
+  Transform = {
+    Translation = {
+      Type = "StaticTranslation",
+      Position = { 3500000, 0, 0 }
+    },
+    Rotation = {
+      Type = "StaticRotation",
+      Rotation = { 0.0, 0.0, -math.pi / 2.0 } -- -90 degrees
+    },
+    Scale = {
+      Type = "StaticScale",
+      Scale = 698000000
+    }
+  },
+  Renderable = {
+    Type = "RenderableSphericalGrid",
+    Size = size,
+    Color = { 1.0, 1.0, 0.0 },
+    LineWidth = 2.0,
+    LongSegments = 2,
+    LatSegments = 64
+  },
+  GUI = {
+    Name = "Carrington Prime Meridian",
+    Path =  "/Solar System/Heliosphere",
+    Description = [[A 2x64 spherical grid showing the Carrington prime meridian]]
+  }
+}
+
+local grid_slice = {
+  Identifier = "WSA_GridSlice",
+  Parent = transforms.HeliocentricEarthEquatorial.Identifier,
+  Transform = {
+    Rotation = {
+      Type = "StaticRotation",
+      Rotation = { 0.0, 0.0, -math.pi / 2.0 } -- -90 degrees
+    },
+    Scale = {
+      Type = "StaticScale",
+      Scale = size
+    }
+  },
+  Renderable = {
+    Type = "RenderableSphericalGrid",
+    Size = size,
+    Color = { 0.8, 0.8, 0.8 },
+    LineWidth = 2.0,
+    LongSegments = 2,
+    LatSegments = 64
+  },
+  GUI = {
+    Name = "Solar longitude facing the Earth",
+    Path =  "/Solar System/Heliosphere",
+    Description = [[An arch on the Sun suface that always faces Earth]]
+  }
+}
+
+local grid_10deg = {
+  Identifier = "WSA_Grid10Degrees",
+  Parent = transforms.HeliocentricEarthEquatorial.Identifier,
+  Transform = {
+    Scale = {
+      Type = "StaticScale",
+      Scale = size
+    }
+  },
+  Renderable = {
+    Type = "RenderableSphericalGrid",
+    Size = size,
+    Color = { 0.035, 0.675, 0.255 },
+    LineWidth = 1.0,
+    Segments = 36
+  },
+  GUI = {
+    Name = "Grid on Sun",
+    Path =  "/Solar System/Heliosphere",
+    Description = [[A grid aligned with the Sun-Earth line.
+      10 degree seperationg between line segments on default.]]
```

Comment:
Looks like a word is missing in the beginning of the second sentence (for flow), and there'sa typo. Maybe: 

 ```suggestion
    Description = [[A grid aligned with the Sun-Earth line, with a 10-degree separation between
        line segments by default.]]
```